devise-security.gemspec

```diff
@@ -39,6 +39,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'pry-rescue'
   s.add_development_dependency 'pry'
   s.add_development_dependency 'rails_email_validator'
-  s.add_development_dependency 'rubocop', '~> 0.58.0'
+  s.add_development_dependency 'rubocop'
```

Comment:
I actually think we should lock this to the same version as the codeclimate channel to ensure they stay in sync.